general/error.cpp

```diff
@@ -153,6 +153,8 @@ void mfem_error(const char *msg)
       mfem::err << "\n\n" << msg << "\n";
    }
 
+   std::abort(); // force crash by calling abort
```

Comment:
I think this should be removed (the correct place for this is at the end of the function, otherwise libunwind, exceptions, etc. won't work).